### ISSUE TITLE
Advocacy: Make a video about Amplication.

### ISSUE THREAD

**Issue by souravjain540** [amplication/amplication#4007]
We always promote non and low-code contributions to Amplication.

In this issue, you have to make a video about the experience of using Amplication, or you can make a tutorial doing any project using Amplication.

This video should be published on any platform of your choice(for example, Youtube, Twitch, etc.).

You have to create a PR by making a new file(in which there is link to that video) in `tutorials` folder.

**Comment by VoidCupboard** [comment]
Can I work on this issue :3 
I would love to get some ideas about which app should I create for showing the demo.
(Since my previous videos got deleted somehow, I am gonna be recreating them soon with a better explanation, so I would love to get some other app ideas rather than those)

**Comment by souravjain540** [comment]
I would love to see your videos. Just make sure for valid contributions they are not deleted.

you can start working on them, and reach out to me for any help :) @VoidCupboard

**Comment by VoidCupboard** [comment]
Yeah ikr, I'm gonna create a backup in the gcloud as well just to make sure that I have a copy of the edited video :P

**Comment by munyoudoum** [comment]
@souravjain540 I used to make a youtube video about it https://www.youtube.com/watch?v=_BeWDanPtZg ... I'm not really a professional youtuber or anything, and just wanted to explore contents creation. Is this good enough for the PR? 😄

**Comment by yuval-hazaz** [comment]
> @souravjain540 I used to make a youtube video about it https://www.youtube.com/watch?v=_BeWDanPtZg ... I'm not really a professional youtuber or anything, and just wanted to explore contents creation. Is this good enough for the PR? 😄

@munyoudoum that is excellent! Thanks for creating this video.

please send a PR with the instructions above and we will approve it

**Comment by Abubakrce19** [comment]
Can i make a video too?

**Comment by syedzubeen** [comment]
@souravjain540 
Is this issue still accepting contributions ?

**Comment by souravjain540** [comment]
> @souravjain540 
> 
> Is this issue still accepting contributions ?

Yes

**Comment by Gingaaa** [comment]
Hey I have worked on around 7-8 company based project, but never countribute to open source this is my first time can you assign me this task.

**Comment by Anuragwagh** [comment]
Hello, I have experience of making videos, I will create a basic beginner friendly video tutorial on Amplication.

**Comment by craigouma** [comment]
@souravjain540 Is this issue still accepting contributions?
